manifests/art.yaml

```diff
@@ -0,0 +1,16 @@
+updates:
+  - file: "{MAJOR}.{MINOR}/cluster-logging.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+    update_list:
+    # replace metadata.name value
+    - search: "clusterlogging.v{MAJOR}.{MINOR}.0"
+      replace: "clusterlogging.{FULL_VER}"
+    # replace entire version line, otherwise would replace 4.2.0 anywhere
+    - search: "version: 4.2.0"
+      replace: "version: {FULL_VER}"
+    - search: 'olm.skipRange: ">=4.1.0 <4.2.0"'
+      replace: 'olm.skipRange: ">=4.1.0 <{FULL_VER}"'
+  - file: "cluster-logging.package.yaml"
+    update_list:
+    - search: "currentCSV: clusterlogging.v{MAJOR}.{MINOR}.2"
```

Comment:
I think this needs to be `.0` instead of `.2`?

Reply:
yup, thanks, updated.